packages/api/src/contracts/user.repository.ts

```diff
@@ -0,0 +1,12 @@
+import type { RegisteredUser, User, UserId } from '@coderscamp/shared/models';
+
+export const USER_REPOSITORY_PORT = Symbol('UserRepositoryPort');
+export interface UserRepositoryPort {
```

Comment:
It's conceptually a port, but it's not a must to use Port postfix in name. But what it's better for you :) 
Normally every repository is a Port. Even if you're using TypeORM / Prisma those repositories are also Ports over cetrain database implementations.